docs/source/examples/substra_core/diabetes_example/run_diabetes.ipynb

```diff
@@ -0,0 +1,725 @@
+{
```

Comment:
<p>Nor this one, technically</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>